packages/contracts/contracts/test/L1Block.t.sol

```diff
@@ -1,22 +1,20 @@
+//SPDX-License-Identifier: MIT
 pragma solidity 0.8.10;
 
 import { DSTest } from "../../lib/ds-test/src/test.sol";
+import { Vm } from "../../lib/forge-std/src/Vm.sol";
```

Comment:
This change matches the boiler plate used in https://github.com/ethereum-optimism/optimistic-specs/blob/main/packages/contracts/contracts/test/L2OutputOracle.t.sol